packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -243,6 +243,7 @@ const MT5PersonalDetailsForm = ({
                                                     use_text={true}
                                                     onChange={e => setFieldValue('citizen', e.target.value, true)}
                                                     required
+                                                    is_disabled_hidden={false}
```

Comment:
The prop name here doesn't match with the name in SelectNative.

Reply:
thanks @akmal-binary, changed to new name